classes/oc/captcha.php

```diff
@@ -84,7 +84,7 @@ public static function check($name = '', $ajax = FALSE)
         //d(Session::instance()->get('captcha_'.$name));
         //d(Session::instance()->get('captcha_'.$name) == strtolower(core::post('captcha')));
 
-        if (core::config('advertisement.captcha') == FALSE) // Captchas are disabled
+        if (core::config('advertisement.captcha') == FALSE AND core::config('general.captcha') == FALSE) // Captchas are disabled
```

Comment:
This wont work on OE, no? theres no 'advertisement.captcha' config and will return FALSE.

I tihnk just with ```if ( core::config('general.captcha') == FALSE)``` is enough